integrationtests/rtt_test.go

```diff
@@ -59,8 +64,32 @@ var _ = Describe("non-zero RTT", func() {
 
 		Context(fmt.Sprintf("with quic version %d", version), func() {
 			It("gets a file with 10ms RTT", func() {
-				runRTTTest(10*time.Millisecond, version)
+				runRTTTest(10*time.Millisecond, version, 20)
 			})
 		})
 	}
+	version := protocol.Version35
+	Context(fmt.Sprintf("with quic version %d", version), func() {
+		fileSizes := [...]int{dataLen, 5 * 1024 * 1024, dataLongLen}
+		for _, fileSize := range fileSizes {
+			fileSizeMB := fmt.Sprintf("%.2f", float64(fileSize)/1024/1024)
+			It("gets a "+fileSizeMB+"MB file with 75ms RTT", func() {
+				rtt := 75 * time.Millisecond
+				testname := "./large_rtt_" + rtt.String() + "_Q" + fmt.Sprint(version) + "_" + fileSizeMB + "MB.log"
+
+				f, err := os.Create(testname)
```

Comment:
Remove the logging here.

FYI, when running integrationtests, you can run them with `QUIC_GO_LOG_LEVEL=DEBUG gingko -- -logfile=log.txt`, so you don't have to modify the test code when you want a log.